docs/docs/cmd/aad/user/user-add.md

```diff
@@ -0,0 +1,160 @@
+# aad user add
+
+Creates a new user
+
+## Usage
+
+```sh
+m365 aad user add [options]
+```
+
+## Options
+
+`--displayName <displayName>`
+: The name to display in the address book for the user.
+
+`--userName <userName>`
+: The user principal name (someuser@contoso.com).
+
+`--accountEnabled [accountEnabled]`
+: Whether the account is enabled. Possible values: `true`, `false`. Default value is true.
+
+`--mailNickname [mailNickname]`
+: The mail alias for the user. By default this value will be extracted from `userName`.
+
+`--password [password]`
+: The password for the user. When not specified, a password will be generated.
+
+`--firstName [firstName]`
+: The given name (first name) of the user. Maximum length is 64 characters.
+
+`--lastName [lastName]`
+: The user's surname (family name or last name). Maximum length is 64 characters.
+
+`--forceChangePasswordNextSignIn`
+: Whether the user should change his/her password on the next login.
+
+`--forceChangePasswordNextSignInWithMfa`
+: Whether the user should change his/her password on the next login and setup MFA.
+
+`--usageLocation [usageLocation]`
+: A two letter [country code](https://learn.microsoft.com/en-us/partner-center/commercial-marketplace-co-sell-location-codes#country-and-region-codes) (ISO standard 3166). Required for users that will be assigned licenses.
+
+`--officeLocation [officeLocation]` 
+: The office location in the user's place of business.
+
+`--jobTitle [jobTitle]`
+: The user's job title. Maximum length is 128 characters.
+
+`--companyName [companyName]`
+: The company name which the user is associated. The maximum length is 64 characters.
+
+`--department [department]`
+: The name for the department in which the user works. Maximum length is 64 characters.
+
+`--preferredLanguage [preferredLanguage]`
+: The preferred language for the user. Should follow [ISO 639-1 Code](https://learn.microsoft.com/en-us/openspecs/office_standards/ms-oe376/6c085406-a698-4e12-9d4d-c3b0ee3dbc4a). Example: `en-US`.
+
+`--managerUserId [managerUserId]`
+: User ID of the user's manager. Specify `managerUserId` or `managerUserName` but not both.
+
+`--managerUserName [managerUserName]`
+: User principal name of the manager. Specify `managerUserId` or `managerUserName` but not both.
+
+--8<-- "docs/cmd/_global.md"
+
```

Comment:
Let's add a remarks section with an info box which permissions are needed to run this command.